e2e-tests/scripts/ci-build.bat

```diff
@@ -0,0 +1,24 @@
+:: Script specific to internal CI, executes this  on windows VM
+
+cd C:\Users\builder\okta\samples-aspnet-webforms\e2e-tests\
+call npm install
+call npm run pretest
```

Comment:
Do we need the `if errorlevel 1 (SET error=true)` here as well or does an npm error cause it to short-circuit?  Or do we not care?

Reply:
We don't care if pretest fails. If it fails, the `npm run test` will fail anyway, which will be caught later.